balancer/endpointsharding/endpointsharding.go

```diff
@@ -285,6 +285,8 @@ func (bw *balancerWrapper) UpdateState(state balancer.State) {
 	bw.es.updateState()
 }
 
+// ParseConfig parses a child config list and returns a LB config for the
+// gracefulswitch Balancer.
```

Comment:
Don't mention graceful switch here.  It's an implementation detail of this package.  It returns an LB config to use with the endpointsharding balancer.

But we can steal the description of how the config should look from gsb:

```
cfg is expected to be a JSON array of LB policy names + configs as the format of the loadBalancingConfig field in ServiceConfig.
```

Reply:
Removed and copied cfg part from downstream method

Reply:
Can you use the text I suggested instead, please?